seatunnel-connectors-v2/connector-hudi/src/main/java/org/apache/seatunnel/connectors/seatunnel/hudi/source/HudiSource.java

```diff
@@ -1,164 +0,0 @@
-/*
```

Comment:
Any reason for remove hudi source?

Reply:
According to the design principles of the SeaTunnel connector, it should not rely on a specific engine. Hudi source currently must rely on the specified engine. It does not have a Java SDK version, so delete Hudi source for now.

Reply:
what mean about must have a Java version?